invenio_circulation/transitions/transitions.py

```diff
@@ -11,7 +11,11 @@
 from datetime import timedelta
 
 from flask import current_app
+from invenio_db import db
 
+from ..api import get_available_item_for_document, \
```

Comment:
What about importing the `api`? It would be clearer to see where the calls come from `api.get_available()` and we won't have this hairy import thing.